jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannel.java

```diff
@@ -765,9 +765,9 @@ public boolean onEarlyEOF()
     public void onBadMessage(BadMessageException failure)
     {
         int status = failure.getCode();
-        String reason = failure.getReason();
+        String message = failure.getReason();
```

Comment:
I think you should revert the changes in this file, these seem to be related to a different issue. (such as #3225)